src/containers/ToolboxSubject/ToolboxSubjectContainer.tsx

```diff
@@ -230,11 +227,11 @@ ToolboxSubjectContainer.willTrackPageView = (
 };
 
 ToolboxSubjectContainer.getDimensions = (props: Props) => {
-  const { subject, locale, topicList, user } = props;
+  const { subject, topicList, user } = props;
   const topicPath = topicList.map(t =>
     subject.allTopics?.find(topic => topic.id === t),
   );
-  const longName = getSubjectLongName(subject.id, locale);
+  const longName = getSubjectLongName(subject.id);
```

Comment:
Om du ser på definisjonen av getSubjectLongName så returneres undefined dersom locale ikkje sendes inn. Det må vi unngå for ellers så sendes ikkje fagnavnet i trackinga. Så alle plasser der du kaller denne funksjonen må du nesten sende inn enten locale eller i18n.language.